### PR TITLE
Upgrade EVMC to version 7.5.0

### PR DESCRIPTION
**Pull request by chfast**

### CODE REVIEW

File: test/utils/utils.hpp [hunk 1]
```diff
@@ -3,28 +3,27 @@
...
-#include <cstdint>
-#include <string>
+#include <evmc/hex.hpp>
 
 using bytes = std::basic_string<uint8_t>;
 using bytes_view = std::basic_string_view<uint8_t>;
 
-/// Encode a byte to a hex string.
-inline std::string hex(uint8_t b) noexcept
+inline auto hex(uint8_t b)
```
Comment (axic) [inline test/utils/utils.hpp:11]: This is noexcept upstream.

File: test/utils/utils.hpp [hunk 2]
```diff
@@ -3,28 +3,27 @@
...
 #pragma once
 
-#include <cstdint>
-#include <string>
+#include <evmc/hex.hpp>
 
 using bytes = std::basic_string<uint8_t>;
 using bytes_view = std::basic_string_view<uint8_t>;
 
-/// Encode a byte to a hex string.
```
Comment (axic) [inline test/utils/utils.hpp:12]: Nit: the others kept the docs, except this one.

Comment (chfast) [reply test/utils/utils.hpp:12]: Fixed.

File: test/utils/utils.hpp (hunk 1 above)
Comment (chfast) [reply test/utils/utils.hpp]: Fixed.

Comment (axic) [reply test/utils/utils.hpp]: So this function is just not needed?

**Review by axic — APPROVED** [review]

File: test/utils/utils.hpp (hunk 1 above)
Comment (chfast) [reply test/utils/utils.hpp]: This is now done with `using evmc::hex`. Sometimes I forget simple ways of doing things in C++.

Comment (axic) [reply test/utils/utils.hpp]: Ah, missed those lines in the diff.

**Pull request merged**